src/main/resources/l10n/JabRef_en.properties

```diff
@@ -673,6 +673,14 @@ Quit\ JabRef=Quit JabRef
 
 Read\ only=Read only
 
+Rebind\ C-a,\ too=Rebind C-a, too
```

Comment:
I am not an Emacs user, is this the correct terminology? Does an Emacs user understand what is meant with that?

Reply:
Yes! That was the UI of 4.3.1: 

![grafik](https://user-images.githubusercontent.com/1366654/75483201-a2dc2100-59a6-11ea-959d-b1780461cc8c.png)

Reply:
I removed the ", too" as it is a bit confusing. I think the indentation and the automatic enabling/disabling is enough to show that it is part of the Emacs keybindings.